python/packages/isce3/signal/rfi_freq_null.py

```diff
@@ -85,10 +86,15 @@ def run_freq_notch(
         Number of azimuth pulses within the raw data to be processed at once
         It is desirable to divide the raw data into processing blocks
         consisted of dimensions [num_pulses_az x num_samples_rng_blk]
-    num_rng_blks: int, default=1
-        Number of blocks in the range dimension, default=1
-        When num_rng_blks=1, all the range samples of the pulses within a 
-        processing block are used to estimate detection mask.
+    num_samples_rng_blk: int, default=256
+        Number of range samples per range block when data blockin is applied 
+        in range direction. It is recommended that this parameter is at least 
+        5 x cpi_len to avoid SINR degradation. In addition, in order to avoid 

```

Comment:
```suggestion
        5 x cpi_len to avoid SNR degradation. In addition, in order to avoid 
```
Following from your discussion with Heresh below